app.py

```diff
@@ -1,1268 +1,174 @@
 # -*- coding: utf-8 -*-
-# Authors: Nanda H Krishna (https://github.com/nandahkrishna), Naveen Narayanan (https://github.com/naveenggmu), Mahesh Bharadwaj (https://github.com/MaheshBharadwaj)
+# Mahesh Bharadwaj K(https://github.com/MaheshBharadwaj)
```

Comment:
Let's just remove all this, GitHub takes care of attribution and this was only added because the `solarillionfoundation` account used to commit here.